ubuntudesign/www.ubuntu.com
# tidy of scratch.scss

## Done

* moved three of the items in _scratch.scss into the section specific scss files so the are not lost
* removed unused .aubergine-hero class

## QA

1. see that /legal/ubuntu-advantage/service-description .nested-counter still works
2. see that /desktop the fade works on the list of verticals
3. see that /internet-of-things and /internet-of-things/digital-signage the list of logos still works
